### DISCUSSION TITLE
[A-L] (2023/24) Foglio 1 - Esercizio 6

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/calcolo-delle-probabilita#138] [Esercizi - Bertini] [answered]
![Es 6](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/107944476/a04b1c4a-624d-4180-9c4c-e40fb9b2fab2)

**Comment by Elia-Belli** [comment, marked as answer]
# Soluzione confermata in classe
- **1)** Per la proprietà distributiva rispetto all'unione possiamo riscrivere:
 $\mathbb{P}((A\cup B)\cap C) = \mathbb{P}((A \cap C) \cup (B \cap C))$
in quanto funzione di probabilità $\mathbb{P}$ è una funzione additiva, allora:
 $\mathbb{P}((A \cap C) \cup (B \cap C)) =  \mathbb{P}(A \cap C) +  \mathbb{P}(B \cap C) = 1/5 + 2/5 = 3/5$

- **2)** Corretta l'osservazione di @CiottoloMaggico per il caso limite di $\mathbb{P}(A \cap B)=1$
Analizzo l'altro caso limite, ovvero $\mathbb{P}(A \cap B)=0$, siccome $\mathbb{P}( \emptyset)=0$ ciò si traduce con $A \cap B= \emptyset$,
ciò non contraddice l'ipotesi $A \cap B \cap C = \emptyset$ perchè $\emptyset \cap C = \emptyset$.
Nel caso massimo $A \cap B$ contiene tutti gli elementi che rimangono al di fuori di $A \cap C$ e $B \cap C$:
$\mathbb{P}(A \cap B) + \mathbb{P}(A \cap C) + \mathbb{P}(B \cap C) \leq 1$
$\Rightarrow \mathbb{P}(A \cap B) \leq 1- \mathbb{P}(A \cap C) - \mathbb{P}(B \cap C) \rightarrow \mathbb{P}(A \cap B) \leq 1-2/5-1/5 = 2/5$

    Quindi $0 \le \mathbb{P}(A \cap B) \leq 2/5$.

**Reply by CasuFrost** [reply]
Scusa, ma per il punto (1), non dovresti scrivere che :
$\mathbb{P}((A\cap C)\cup (B\cap C))=\mathbb{P}(A\cap C)+\mathbb{P}(B\cap C)-\mathbb{P}(A\cap B\cap C)$
Che poi quest'ultima sia nulla fa coincidere il risultato, ma nella formula non dovrebbe essere incluso?

**Reply by Elia-Belli** [reply]
Si, l'ho omesso perchè per ipotesi sono disgiunti a 3

**Comment by CiottoloMaggico** [comment]
Propongo una soluzione per il punto 2.
Osserviamo che P(Ω) = 1, di conseguenza P(A ∩ B) = 1 vorrebbe dire A ∩ B = Ω ossia che  ∀ω∈Ω vale ω∈A ∩ ω∈B.
Essendo A ∩ B = Ω  allora C ⊆ Ω = A ∩ B, consegue che A ∩ B ∩ C != ∅ il che contraddice la premessa A ∩ B ∩ C == ∅
